packages/excavator-web/etl.ts

```diff
@@ -163,6 +164,9 @@ async function main() {
 
         const { _PROJECT, _VERSION, ...data } = fixed;
 
+        if (projects.find((project) => project.name === _PROJECT)?.completed)
```

Comment:
Honestly this might just a be something we apply in applyFixes. At the very least do it before applyFixes.